scripts/joinmarket-qt.py

```diff
@@ -1434,6 +1434,9 @@ def initUI(self):
         exportPrivAction = QAction('&Export keys', self)
         exportPrivAction.setStatusTip('Export all private keys to a  file')
         exportPrivAction.triggered.connect(self.exportPrivkeysJson)
+        changePassAction = QAction('&Change passphrase', self)
```

Comment:
This should be "&Change passphrase..." now, see #640.

Reply:
Sorry flew over my head thanks!